src/components/calendar/MeetingModal.js

```diff
@@ -43,18 +41,36 @@ const MeetingModal = props => {
 
   const submitMeeting = e => {
     e.preventDefault();
-    let newParticipants = [];
+    console.log("meeting", meeting);
+    let newParticipants = {
```

Comment:
This section was added in order to save an array of names of participants as well as an array of participant UIDs to firebase.